### ISSUE TITLE
How is output resolution taken into account if differs from input resolution?

### ISSUE THREAD

**Issue by danergo** [gyroflow/gyroflow#584]
Hi, Folks!

When I open a video in Gyroflow:

![Screenshot_20230207_072319](https://user-images.githubusercontent.com/11708344/217289344-a87fc8a0-1be3-4243-8642-0523a62ec192.png)

And save it as:

![Screenshot_20230207_072507](https://user-images.githubusercontent.com/11708344/217289387-fd099aa5-6855-4f8b-b282-b2fb6412eb2f.png)

Will it only cut the top,bottom,left,right? Or it also helps for the stabilization? As it's in the export settings, I'm unsure. It might be helpful for the stabilization: if Gyroflow knows the left & right XY pixels, and top/bottom AB pixels are not needed, it could expand its room for the stabilization.

Anyone knows how is this taken into account?

Thank you very much.

**Comment by AdrianEddy** [comment]
It also helps with the stabilization, basically the output resolution is the target rectangle that the video can move around in. You can see that clearly in `Stabilization overview` mode in [latest dev build](https://gyroflow.xyz/devbuild/?autodownload) and you can change the output resolution to see how it behaves.

TLDR: it's not a static crop, is helps with stabilization
